provision/acc_provision/templates/aci-operators-crd.yaml

```diff
@@ -10,4 +10,22 @@ spec:
     plural: acicontainersoperators
     singular: acicontainersoperator
   scope: Namespaced
-  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        description: CRD to own the lifecycle of ACI objects in the cluster
+        properties:
+          spec:
+            description: Spec required for a ACI CNI CRD
+            properties:
+              flavor:
+                type: string
+              config:
+                type: string
+            type: object
+        required:
+        - spec
```

Comment:
Question: How this field ( - spec) is considered as required?  Is it because we analyze the spec field.

Reply:
I added it as a required field assuming we need it to be there, is that correct?

Reply:
I need to go and check too. Let's discuss this offline

Reply:
It's correct.  Marking as resolved